entities/StandardBoard.cs

```diff
@@ -0,0 +1,176 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace LightChess
+{
+
+    public class StandardBoard: IBoard
+    {
+        /*
+            The class represents the board containing piece's locations/cells. 
+            The width means A-H, while height - 1-8.
+        */
+
+        public int Width = 8;
+        public int Height = 8;
+
+        // Dictionary to save pieces by cell.
+        public Dictionary<Cell, Piece> PositionToPiece;
+
+        public StandardBoard()
+        {
+            PositionToPiece = new Dictionary<Cell, Piece>();
+        }
+
+        public StandardBoard(Dictionary<Cell, Piece> positionToPiece)
+        {
+            // Deep copy of dictionary.
+            PositionToPiece = positionToPiece.ToDictionary(entry => entry.Key,
```

Comment:
What did you mean by "Deep copy" here exactly?

I can see that you make a shallow copy as objects used for keys and values reused. And that's fine given that the object are immutable as I understand.

Another way to achieve the same is:
`PositionToPiece = new Dictionary<Cell, Piece>(positionToPiece);`

Reply:
Fixed!